src/Middleware/Rewrite/src/ApacheModRewrite/FlagParser.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Microsoft.AspNetCore.Rewrite.ApacheModRewrite;
 
-internal sealed class FlagParser
+internal static class FlagParser
```

Comment:
This was just silly, both the class and the `_ruleFlagLookup` dictionary were allocated every time this type was used.